shared/libraries/config_protocol/tests/test_config_client_server.cpp

```diff
@@ -560,8 +555,10 @@ TEST_F(ConfigProtocolTest, DeviceGetAvailableDevices)
         .WillOnce(Return(device.ptr.asPtr<IComponent>()));
 
     EXPECT_CALL(device.mock(), getAvailableDevices(_)).WillOnce(Return(OPENDAQ_SUCCESS));
+    */
 
-    client->getClientComm()->sendComponentCommand("/dev", "GetAvailableDevices");
+    // this test should probaly be changed to call client->connect() first, so protocol version is correctly negotiated
+    ASSERT_THROW(client->getClientComm()->getAvailableDevices("/dev"), ServerVersionTooLowException);
```

Comment:
@dejancrn what steps do I need to take, so I will be able to call connect() method in this tests?